app/assets/stylesheets/modal.scss

```diff
@@ -4,4 +4,23 @@ button {
   &.active {
     border: 0.25em solid $primary-color;
   }
+  &.active::before {
```

Comment:
<img width="555" alt="a screenshot of the modal trigger toggle buttons. checkmark prefixes the one that starts with 'motion modal'" src="https://user-images.githubusercontent.com/4480480/115777614-2317cf80-a37b-11eb-969d-18effc5306e2.png">